samples/HealthChecksSample/GCInfoHealthCheck.cs

```diff
@@ -65,8 +65,8 @@ public Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, Canc
                 { "Gen2Collections", GC.CollectionCount(2) },
             };
 
-            // Report failure if the allocated memory is >= the threshold
-            var result = allocated >= options.Threshold;
+            // Report failure if the allocated memory is >= the threshold. Negated because true == success
+            var result = !(allocated >= options.Threshold);
```

Comment:
Or you know `allocated < options.Threshold`

Reply:
#TheTaylor